specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -153,10 +164,44 @@ Consider at least the below topics but you do not need to cover those that are n
       * Is it logged?
   * What secrets are needed by the components? How are these secrets secured and attained?
 -->
-### CICD
+### CI/CD
 
-#### e2e test image
-To run e2e tests alot of dependencies are needed (browsers, build systems etc...).
+CI/CD for frontend components leverages existing CI/CD for the VDK monorepo using
+GitLab.
+
+- [GitLab config for frontend components](/projects/frontend/cicd/.gitlab-ci.yml)
+- [@versatiledatakit/shared build script](/projects/frontend/cicd/build_shared.sh)
+- [@versatiledatakit/data-pipelines build script](/projects/frontend/cicd/install_data_pipelines.sh)
```

Comment:
nit: so the install script performs install and build, and what does the build script perform?
It just seems non-uniform at first sight